css/main.css

```diff
@@ -1,3 +1,96 @@
-/* Add your custom CSS here */
-
-/* Dont' forget to link this file to your HTML in the <head> */
+body {
+    color: #2F2F2E
+}
+.buttons {
+    background-color: #CFF4EC;
+    border: #CFF4EC;
+    padding: 10px 30px 10px 30px;
+    margin: 30px 0 20px 10px;
+    font-family: 'Quicksand', sans-serif;
+}
+.buttons:hover {
+    background-color: #4D7369;
+    color: white;
+}
+#booking{
+    float: right
+}
```

Comment:
Please don't use `float` rules in your CSS. Float is the old way of making layouts in CSS and it has now been replaced by Flexbox (which was introduced a few years ago). 
Set `display: flex` on the parent if you want to control where this element sits.